src/mail/view/MailViewModel.ts

```diff
@@ -138,12 +138,13 @@ export class MailViewModel {
 		this.singInit()
 		const conversationEnabled = this.conversationPrefProvider.getConversationViewShowOnlySelectedMail()
 		if (this.conversationViewModel && this.conversationPref !== conversationEnabled) {
+			const mail = this.conversationViewModel.primaryMail
 			this.createConversationViewModel({
-				mail: this.conversationViewModel.primaryMail,
+				mail,
 				showFolder: false,
-				// FIXME
 				delayBodyRenderingUntil: Promise.resolve(),
 			})
+			this.mailOpenedListener.onEmailOpened(mail)
```

Comment:
just from the diff it's not clear why moving the callback call out of .createConversationViewModel fixes the multiple calls